docker-compose.yml

```diff
@@ -16,6 +16,21 @@ services:
       AWS_ACCESS_KEY_ID: ${CKAN_AWS_ACCESS_KEY_ID}
       GH_Token: ${CKAN_GH_Token}
       SQS_QUEUE: OutboundDev.fifo
+      SQS_TIMEOUT: 30
+    volumes:
+      - ./netkan:/home/netkan/netkan
+  scheduler:
+    build:
+      dockerfile: scheduler/Dockerfile
+      context: .
+      target: dev
+    environment:
+      NETKAN_PATH: ${NETKAN_METADATA_PATH}
+      AWS_DEFAULT_REGION: ${CKAN_AWS_DEFAULT_REGION}
+      AWS_SECRET_ACCESS_KEY: ${CKAN_AWS_SECRET_ACCESS_KEY}
+      AWS_ACCESS_KEY_ID: ${CKAN_AWS_ACCESS_KEY_ID}
+      SQS_QUEUE: InboundDev.fifo
+      MAX_QUEUED: 1
```

Comment:
The commit history says this line was added for dev testing. Should it be updated or removed now?

Reply:
You're likely missing a piece of information that only exists in my head right now. The docker compose is for development testing only. There are legit ways to use a compose file in prod, but we're already well tied to AWS and I know Elastic Container Service really well, so short of a provider bankrolling the infrastructure with some long term guarantees, I think it's a safe enough option for now (it's also essentially "free", as you only pay the ec2 costs).

Reply:
> The docker compose is for development testing only.

Ahh, OK. Never mind!

Reply:
Good to ask though! If something doesn't make sense, it's either locked in my brain or it properly doesn't make sense (the latter happens more than I'd like to admit :smile: )